internal/s3manager/s3manager.go

```diff
@@ -126,6 +135,15 @@ func GetConnectionDetails(ctx context.Context, cluster *kubernetes.Cluster, secr
 	}
 	details.Bucket = string(secret.Data["bucket"])
 
+	// load ca cert from minio-tls
+	// if secret is missing just log and skip
+	secret, err = cluster.GetSecret(ctx, "epinio", "minio-tls")
```

Comment:
for now it's fine but this should be an argument we pass to the epinio server. Maybe the user deployed some S3 compatible solution and they have another secret we should trust instead.

Reply:
Also epinio server shouldn't have hardcoded minio references. It shouldn't care what S3 it's talking to.

Reply:
added a setting in values.yaml in the helm chart and an argument in `epinio server` command here: https://github.com/epinio/epinio/pull/1228/commits/ab0c4b71dffcf53a1fe0fcb8fe98bcf21dc88cde